tests/RobotFramework/tests/cumulocity/supported_operations/mapper-publishing-agent-supported-ops.robot

```diff
@@ -0,0 +1,52 @@
+*** Settings ***
+Resource    ../../../../resources/common.resource
+Library    Cumulocity
+Library    ThinEdgeIO
+
+Test Tags    theme:c8y    theme:operation    theme:tedge-agent
+Test Setup    Custom Setup
+Test Teardown    Get Logs
+
+*** Test Cases ***
+Publish the tedge agent supported operations delete restart
+    # wait till list request is pushed out
+    Should Have MQTT Messages    tedge/commands/req/software/list
+    # stop mapper and remove the supported operations
+    Execute Command    sudo systemctl stop tedge-mapper-c8y
```

Comment:
You can use `Stop Service` key instead of `Execute Command`. (Also for restart, start, ...)

Reply:
Yes, done.